src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -7183,5 +7183,934 @@ static void F()
+            var compilation1 = compilation0.WithSource(source1.Tree);
+            var compilation2 = compilation1.WithSource(source2.Tree);
+
+            var f0 = compilation0.GetMember<MethodSymbol>("C.F");
+            var f1 = compilation1.GetMember<MethodSymbol>("C.F");
+            var f2 = compilation2.GetMember<MethodSymbol>("C.F");
+
+            var v0 = CompileAndVerify(compilation0);
+            v0.VerifyIL("C.F", @"
+{
+  // Code size       17 (0x11)
+  .maxstack  2
+  .locals init (int V_0, //x
+                int V_1, //y
+                int V_2, //z
+                int V_3)
+  IL_0000:  nop
+  IL_0001:  ldc.i4.1
+  IL_0002:  stloc.0
+  IL_0003:  ldc.i4.2
+  IL_0004:  stloc.1
+  IL_0005:  ldc.i4.3
+  IL_0006:  stloc.2
+  IL_0007:  ldloc.0
+  IL_0008:  ldloc.1
+  IL_0009:  add
+  IL_000a:  ldloc.2
+  IL_000b:  add
+  IL_000c:  stloc.3
+  IL_000d:  br.s       IL_000f
+  IL_000f:  ldloc.3
+  IL_0010:  ret
+}
+");
+
+            var md0 = ModuleMetadata.CreateFromImage(v0.EmittedAssemblyData);
+
+            var generation0 = EmitBaseline.CreateInitialBaseline(md0, v0.CreateSymReader().GetEncMethodDebugInfo);
+            var diff1 = compilation1.EmitDifference(
+                generation0,
+                ImmutableArray.Create(
+                    new SemanticEdit(SemanticEditKind.Update, f0, f1, GetSyntaxMapFromMarkers(source0, source1), preserveLocalVariables: true)));
+
+            diff1.VerifyIL("C.F", @"
+{
+  // Code size       15 (0xf)
+  .maxstack  2
+  .locals init (int V_0, //x
+                [int] V_1,
+                int V_2, //z
+                [int] V_3,
+                int V_4)
+  IL_0000:  nop
+  IL_0001:  ldc.i4.1
+  IL_0002:  stloc.0
+  IL_0003:  ldc.i4.3
+  IL_0004:  stloc.2
+  IL_0005:  ldloc.0
+  IL_0006:  ldloc.2
+  IL_0007:  add
+  IL_0008:  stloc.s    V_4
+  IL_000a:  br.s       IL_000c
+  IL_000c:  ldloc.s    V_4
+  IL_000e:  ret
+}
+");
+
+            var diff2 = compilation2.EmitDifference(
+                diff1.NextGeneration,
+                ImmutableArray.Create(
+                    new SemanticEdit(SemanticEditKind.Update, f1, f2, GetSyntaxMapFromMarkers(source1, source2), preserveLocalVariables: true)));
+
+            diff2.VerifyIL("C.F", @"
+{
+  // Code size       21 (0x15)
+  .maxstack  2
+  .locals init (int V_0, //x
+                [int] V_1,
+                int V_2, //z
+                [int] V_3,
+                [int] V_4,
+                int V_5, //y
+                int V_6)
+  IL_0000:  nop
+  IL_0001:  ldc.i4.1
+  IL_0002:  stloc.0
+  IL_0003:  ldc.i4.2
+  IL_0004:  stloc.s    V_5
+  IL_0006:  ldc.i4.3
+  IL_0007:  stloc.2
+  IL_0008:  ldloc.0
+  IL_0009:  ldloc.s    V_5
+  IL_000b:  add
+  IL_000c:  ldloc.2
+  IL_000d:  add
+  IL_000e:  stloc.s    V_6
+  IL_0010:  br.s       IL_0012
+  IL_0012:  ldloc.s    V_6
+  IL_0014:  ret
+}
+");
+        }
+
+        [Fact]
+        public void Tuple_Var()
```

Comment:
>public void Tuple_Var( [](start = 8, length = 22)

I don;t think this test is necessary. It's the same as Tuple_Parenthesized from EnC point of view.